frontend/src/store/actions/companies-actions.ts

```diff
@@ -10,11 +12,12 @@ export const GET_COMPANY_BY_ID = 'GET_COMPANY_BY_ID';
 export const UPDATE_COMPANY = 'UPDATE_COMPANY';
 export const DELETE_COMPANY = 'DELETE_COMPANY';
 
+type AppThunkAction<T> = ThunkAction<T, RootStateType, undefined, CompanyActionType | ErrorsActionType>

```

Comment:
her ville jeg nok sendt inn 2 generics, så kan slipper man å ha en stor eller-type og har bedre typing per action
```suggestion
type AppThunkAction<R, A> = ThunkAction<R, RootStateType, undefined, A>

```